packages/terra-list/src/SingleSelectList.jsx

```diff
@@ -129,7 +136,11 @@ class SingleSelectList extends React.Component {
       newProps.onKeyDown = onKeyDown;
     }
 
+    // Uses the props.hasChevron value, unless the consumer specifies the item hasChevron attribute as false.
     newProps.hasChevron = this.props.hasChevrons;
+    if (item.props.hasChevron === false) {
```

Comment:
For chevron we might need to check for undefined, since the value for hasChevrons can be true and false, rather than the isSelectable which is always true.